desktop/core/src/desktop/templates/scheduler/submit_job_popup.mako

```diff
@@ -163,7 +164,17 @@ else:
       else if (_el.val().indexOf("T") == "-1") {
         _el.val(_el.val() + "T" + _val.split("T")[1]);
       }
+
+      $(document).off("click.hideDatepicker");
+    }).on("show", function() {
+      $(document).on("click.hideDatepicker", function (event) {
+        if (!$(event.target).closest(".datepicker, .calendar-link, input[type='text']").length) {
+          _el.datepicker('hide');
+        }
+      });
+      $(document).off("click.hideDatepicker");
```

Comment:
This will remove the event handler right after it's been added on line 170. It's probably not what you intend.